platform/mds/mds/src/main/java/org/motechproject/mds/dto/DraftData.java

```diff
@@ -21,6 +21,7 @@
     public static final String FIELD = "field";
 
     public static final String ADD_NEW_INDEX = "$addNewIndex";
+    public static final String ADD_NEW_FIELD = "indexes.0.$addField";
```

Comment:
do not add constant with hardcoded path for adding new field to the first lookup from the list. I see that you using this only in test, just use this as a raw string in test.